packages/@aws-cdk/aws-stepfunctions-tasks/lib/lambda/invoke.ts

```diff
@@ -85,6 +86,13 @@ export class LambdaInvoke extends sfn.TaskStateBase {
       throw new Error('Task Token is required in `payload` for callback. Use JsonPath.taskToken to set the token.');
     }
 
+    if (this.props.payloadResponseOnly &&
+      (this.integrationPattern || this.props.invocationType || this.props.clientContext || this.props.qualifier)) {
+      throw new Error(
+        'payloadResponseOnly property conflicts with integrationPattern, invocationType, clientContext, and qualifier.',
```

Comment:
```suggestion
        "The 'payloadResponseOnly' property cannot be used if 'integrationPattern', 'invocationType', 'clientContext' or 'qualifier' are specified.",
```

rationale: just a small tweak to be more prescriptive about how to reconcile the error

Reply:
That looks good. I'll update the language in the documentation, too.